docs/network.md

```diff
@@ -206,6 +206,8 @@ The following proposal types are currently supported:
 
 - `upgrade <descriptor.json>`: Network upgrade proposal. Provide a JSON file
   containing the upgrade descriptor.
+- `parameter-change <module-name> <changes.json>`: Network parameter change
+  proposal. Provide the module name and the consensus parameter changes JSON.
```

Comment:
Is there a place where all possible module names are listed?

Reply:
These are the consensus modules. Not sure if there's a list somewhere. Idk how useful even would be listing that, someone creating a parameter change proposal needs somewhat in-depth knowledge of the oasis-core code anyway.